portfolio/src/main/java/com/google/sps/Secrets.java

```diff
@@ -18,36 +18,68 @@
 import com.google.cloud.secretmanager.v1.SecretVersionName;
 import java.io.IOException;
 
+/**
+ * Provides easy access to secrets (like API Keys) after loading them with
+ * {@link com.google.cloud.secretmanager.v1.SecretManagerServiceClient}.
+ */
 public class Secrets {
 
   private static final String PROJECT_ID = "nwach-step-2020";
   private static final String VERSION_LATEST = "latest";
   private static final String PERSPECTIVES_API_KEY_ID = "Perspectives_API_Key";
 
-  public static final String PERSPECTIVES_API_KEY = accessSecret(PERSPECTIVES_API_KEY_ID);
+  private static Secrets INSTANCE;
```

Comment:
```suggestion
  private static Secrets instance;
```

Since its not a constant.